core/src/main/resources/reference.conf

```diff
@@ -21,7 +21,7 @@ akka-persistence-jdbc {
   # This is necessary because we must keep track of the highest sequence number that was ever used on a
```

Comment:
akka-persistence-jdbc - to akka-persistence-pg ?

Reply:
or akka-persistence-postgres

Reply:
I'll change it to akka-persistence-postgres, according to the new plugin name. But this is going to be part of a rebranding story.